content/ember-released-5-3.md

```diff
@@ -0,0 +1,128 @@
+
+### Changes in EmberData 5.3
+
+#### Bug Fixes
+
+EmberData 5.3 introduced 10 bug fixes. Details are in the
+[EmberData 5.3.0 release page](https://github.com/emberjs/data/releases/tag/v5.3.0).
+
+#### Features
+
+EmberData 5.3 introduced 3 features.
+
+This release primarily focused on polish for the `RequestManager` story, introducing builder utils, builders for `REST` `ActiveRecord` and `JSON:API` requests as well as serialization utils for `JSON:API` requests.
+
+As part of shipping serialization utils, we implemented several long-awaited cache features to support them:
+
+- `cache.changedRelationships` retrieves a diff of what has been added or removed from each relationship for a resource
+- `cache.hasChangedRelationships` answers whether any relationships have changed for a resource
+- `cache.rollbackRelationships` will reset all relationships for that resource to their "remote" state, removing any local changes.
+
+#### Deprecations
+
+EmberData 5.3 introduced 5 deprecations.
+
+As part of our ongoing efforts to modernize the library and to support the above changes, we implemented several new deprecations.
+
+- Deprecate legacy import paths (`ember-data/`) except for `ember-data/store`
+- Deprecate non-strict-type (e.g. "users" or "myType" instead of "user" and "my-type")
+- Deprecate non-strict-id (e.g . `1` instead of `"1"`)
+
+And in relationships specifically:
+
+- deprecate fixing non-unique relationship payloads (when the same record is referenced multiple times in a related collection)
+- deprecate remote updates clearing local changes (an upcoming blog post will explain the new behavior here in detail)
+
+For more details on changes in EmberData 5.3, please review the
+[EmberData 5.3.0 release page](https://github.com/emberjs/data/releases/tag/v5.3.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI using the `ember-cli-update` project:
+
+```bash
+npx ember-cli-update
+```
+
+This utility will help you to update your app or addon to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the GitHub README](https://github.com/ember-cli/ember-cli-update).
+
+While it is recommended to keep Ember CLI versions in sync with Ember and EmberData, this is not required. After updating ember-cli, you can keep your current version(s) of Ember or EmberData by editing `package.json` to revert the changes to the lines containing `ember-source` and `ember-data`.
+
+### Changes in Ember CLI 5.3
+
+#### Bug Fixes
+
+Ember CLI 5.3 introduced 3 bug fixes.
+
+
+#### Deprecations
+
+Ember CLI 5.3 introduced 1 deprecation.
+
+- Deprecate outputPaths build option, as output paths will no longer be predetermined under Embroider
```

Comment:
Can we review / merge the guide as well => https://github.com/ember-learn/deprecation-app/pull/1337 ?

Reply:
Done ✅